.github/workflows/ci.yml

```diff
@@ -158,23 +160,6 @@ jobs:
       - run: python -m pip install --upgrade wheel 'tox<4'
       - run: python -m tox -e pyright
 
-  package:
```

Comment:
@hynek is there a reason you don't want to run tests on the sdist?
Would you be open to trying out my new(ish) action? https://github.com/re-actors/checkout-python-sdist — it's essentially a mechanism to use an sdist instead of a Git checkout. This should bring more reliability + ensure the sdists are downstream-friendly.

Reply:
it's on my radar, but i wanted to wait for it to stabilize a bit before embracing it

i'd be open to use it in one of my smaller projects like environ-config i guess

Reply:
Ah, okay. I'm not planning on changing the public API there: I've started it as a part of the experiment to make an action (and later, a reusable workflow), I was using it through the `@master` version and at some point just decided to tag it properly with probably a year of zero changes. Plus it's quite simple, implementation-wise.